src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/Renamer.cs

```diff
@@ -65,6 +65,12 @@ public void HandleRename(string oldFilePath, string newFilePath)
 
         internal async Task HandleRenameAsync(string oldFilePath, string newFilePath)
         {
+            // Do not offer to rename the file in VS Online scenarios.
+            if (Shell.KnownUIContexts.CloudEnvironmentConnectedContext.IsActive)
```

Comment:
Is there an issue tracking enabling the scenario?

Reply:
https://devdiv.visualstudio.com/devdiv/_queries/edit/977305